app/forms/validators.py

```diff
@@ -86,57 +98,58 @@ class NumberRange:
 
     def __init__(
         self,
-        minimum=None,
-        minimum_exclusive=False,
-        maximum=None,
-        maximum_exclusive=False,
-        messages=None,
-        currency=None,
+        minimum: NumType = None,
```

Comment:
Minor but as per our style guide, we should wrap any types initialised as `None` in `Optional`. There are a couple of occurrence like this which we should look to make consistent. https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/doc/python-type-hinting.md#variables